From d54719af52c66e89509b9ee01cfe700bc05be5be Mon Sep 17 00:00:00 2001
From: Mike Primm <mike@primmhome.com>
Date: Thu, 4 Jul 2013 00:03:42 -0500
Subject: [PATCH] Create CraftHorse Instance for EntityHorse

Handle creating instance of CraftHorse for EntityHorse instances and spawn()

diff --git a/src/main/java/org/bukkit/craftbukkit/CraftWorld.java b/src/main/java/org/bukkit/craftbukkit/CraftWorld.java
index a68169f..7c0a026 100644
--- a/src/main/java/org/bukkit/craftbukkit/CraftWorld.java
+++ b/src/main/java/org/bukkit/craftbukkit/CraftWorld.java
@@ -893,6 +893,8 @@ public class CraftWorld implements World {
                 entity = new EntityCreeper(world);
             } else if (Ghast.class.isAssignableFrom(clazz)) {
                 entity = new EntityGhast(world);
+            } else if (Horse.class.isAssignableFrom(clazz)) {
+                entity = new EntityHorse(world);
             } else if (Pig.class.isAssignableFrom(clazz)) {
                 entity = new EntityPig(world);
             } else if (Player.class.isAssignableFrom(clazz)) {
-- 
1.8.1.2

